spyder/plugins/editor/widgets/codeeditor/lsp_mixin.py

```diff
@@ -1293,10 +1293,14 @@ def apply_code_folding(self, folding_info):
         self.highlight_folded_regions()
 
         # Update indent guides, which depend on folding
-        if self.indent_guides._enabled and len(self.patch) > 0:
+        if self.indent_guides._enabled:
             line, column = self.get_cursor_line_column()
             self.update_whitespace_count(line, column)
 
+            # This is necessary to repaint guides in cloned editors and the
+            # original one after making edits in any one of them.
```

Comment:
Maybe it could be worthy to add a reference to the issue here? So something like:

```suggestion
            # original one after making edits in any one of them.
            # See spyder-ide/spyder#23297
```

Reply:
Thanks @dalthviz. I think this is a good idea, so I'm going to apply it.